verificarlo.in.in

```diff
@@ -98,7 +99,7 @@ def linker_mode(sources, options, output, args):
     extra_args = "-static " if args.static else "-fPIC "
     extra_args += "-DINST_FCMP " if args.inst_fcmp else ""
     extra_args += "-DDDEBUG " if args.ddebug else ""
-    shell('{clang} -c -O3 {extra_args} -o .vfcwrapper.o {vfcwrapper} -I {mcalib_includes}'.format(
+    shell('{clang} -c -Wno-varargs -O3 {extra_args} -o .vfcwrapper.o {vfcwrapper} -I {mcalib_includes}'.format(
```

Comment:
It is `-Wno-varags` necessary? It's always nice to have warnings.

Reply:
We discussed this one with @Avaquet. The way we use varargs in enter / exit callbacks raises a false-positive warning. If you do not like the flag, another possibility would be disabling the warning locally with an annotation.